DLM_code/Src/dlm-manage_data_aquisition.c

```diff
@@ -112,8 +112,24 @@ void set_bucket_size(U8 sending_dam, void* UNUSED,
         above_bucket_node->next = bucket_node;
     }
 
-    // set the number of parameters in this bucket
+    // set the number of parameters in this bucket. No parameters have been added yet
+    bucket_node->bucket.params_added = 0;
```

Comment:
Is this intended? The comment and the old line seem to indicate this shouldn't be zero, though I haven't read the while file yet lol

Reply:
There are two values in the bucket, number of parameters currently added and number of parameters that need to be to make it complete